test/async-hooks/test-graph.tcp.js

```diff
@@ -17,7 +17,7 @@ const server = net
 
 server.listen(common.PORT);
 
-net.connect({ port: server.address().port, host: server.address().address },
+net.connect({ port: server.address().port, host: '::1' },
```

Comment:
I don't know from where this commit appeared.It might be related to the "allow edits from maintainers" option on the pull request. Or maybe this commit was deleted from master, but is still in my branch?
It also affects #15013